src/displayapp/screens/WatchFaceLCARS.cpp

```diff
@@ -0,0 +1,345 @@
+  heartbeatIcon = label_make(stepValue, -25, 0, orangeColor, LV_ALIGN_OUT_LEFT_MID, "");
+  lv_obj_align(heartbeatIcon, heartbeatValue, LV_ALIGN_IN_BOTTOM_LEFT, -25, 0); // TODO: Find out why this align is necessary
+
+  // Tasks
+  taskRefresh = lv_task_create(RefreshTaskCallback, LV_DISP_DEF_REFR_PERIOD, LV_TASK_PRIO_MID, this);
+  Refresh();
+}
+
+WatchFaceLCARS::~WatchFaceLCARS() {
+  // Tasks
+  lv_task_del(taskRefresh);
+  
+  // Fonts
+  if (font_antonio_78 != nullptr) {
+    lv_font_free(font_antonio_78);
+  }
+  if (font_antonio_33 != nullptr) {
+    lv_font_free(font_antonio_33);
+  }
+  if (font_antonio_21 != nullptr) {
+    lv_font_free(font_antonio_21);
+  }
+  if (font_antonio_13 != nullptr) {
+    lv_font_free(font_antonio_13);
+  }
+  
+  // Objects
+  lv_obj_clean(lv_scr_act());
+}
+
+void WatchFaceLCARS::Refresh() {
+  currentNanoSeconds = std::chrono::time_point_cast<std::chrono::nanoseconds>(dateTimeController.CurrentDateTime());
+  if (currentNanoSeconds.IsUpdated()) {
+    UpdateSeconds();
+    UpdateStepCount();
+    UpdateBatteryPercent();
+    UpdateBLE();
+    UpdateHeartRate();
+    currentDateTime = std::chrono::time_point_cast<std::chrono::minutes>(currentNanoSeconds.Get());
+    if (currentDateTime.IsUpdated()) {
+      UpdateTime();
+      currentDate = std::chrono::time_point_cast<days>(currentDateTime.Get());
+      if (currentDate.IsUpdated()) {
+        UpdateStardate();
+        UpdateWK();
+      }
+    }
+  }
+}
+
+void WatchFaceLCARS::UpdateHeartRate() {
+  heartbeat = heartRateController.HeartRate();
+  heartbeatRunning = heartRateController.State() != Controllers::HeartRateController::States::Stopped;
+  if (heartbeat.IsUpdated() || heartbeatRunning.IsUpdated()) {
+    if (heartbeat.Get() > 120) {
+      set_label_color(heartbeatValue, redColor);
+      set_label_color(heartbeatIcon, redColor);
+    } else {
+      set_label_color(heartbeatValue, orangeColor);
+      set_label_color(heartbeatIcon, orangeColor);
+    }
+    if (heartbeatRunning.Get()) {
+      lv_label_set_text_fmt(heartbeatValue, "%d", heartbeat.Get());
+      lv_label_set_text_static(heartbeatIcon, Symbols::heartBeat);
+    } else {
+      lv_label_set_text_static(heartbeatValue, "");
+      lv_label_set_text_static(heartbeatIcon, "");
+    }
+    lv_obj_realign(heartbeatValue);
+    lv_obj_realign(heartbeatIcon);
+  }
+}
+
+void WatchFaceLCARS::UpdateTime() {
+  uint8_t hour = dateTimeController.Hours();
+  uint8_t minute = dateTimeController.Minutes();
+
+  if (settingsController.GetClockType() == Controllers::Settings::ClockType::H12) {
+    char ampmChar[3] = "AM";
+    if (hour == 0) {
+      hour = 12;
+    } else if (hour == 12) {
+      ampmChar[0] = 'P';
+    } else if (hour > 12) {
+      hour = hour - 12;
+      ampmChar[0] = 'P';
+    }
+    lv_label_set_text(labelTimeAmPm, ampmChar);
+    weekNumberFormat = "%V";
+  } else {
+    weekNumberFormat = "%U";
+  }
```

Comment:
Whats the reasoning for 12h → Week starts at Monday (%V), 24h → Week starts at Sunday (%U)?
I personally prefer 24h format with a Week that starts at Monday …